src/main/java/seedu/address/model/person/Deadline.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Represents a Person's deadline in the address book.
+ */
+public class Deadline {
+    public static final String MESSAGE_CONSTRAINTS = "Deadlines can only take dd/mm/yyyy";
+
+    public final String value;
+
+    /**
+     * Constructs a {@code Deadline}.
+     *
+     * @param dateAsString a valid date.
+     */
+    public Deadline(String dateAsString) {
+        requireNonNull(dateAsString);
+        checkArgument(isValidDeadline(dateAsString));
+        if (dateAsString.equals("*No deadline specified*")) {
+            new Deadline();
+        }
+        value = dateAsString;
+    }
+
+    public Deadline() {
+        value = "*No deadline specified*";
```

Comment:
I suggest declaring this `"*No deadline specified*"` as a class-level member named something like `NO_DEADLINE_PLACEHOLDER`, to avoid problem of magic literal, and ease of changing in the future, as I notice it might be used a lot. What do you think?

Reply:
Okay, I think this is good.